src/blenderbim/blenderbim/core/qto.py

```diff
@@ -21,3 +21,22 @@ def calculate_circle_radius(qto, obj=None):
     result = qto.get_radius_of_selected_vertices(obj)
     qto.set_qto_result(result)
     return result
+
+def assign_pset_qto(qto, selected_objects):
+    for object in selected_objects:
```

Comment:
Don't use "object" as a variable name in Python

Reply:
Ook! I'll do a PR with this and other fixes.